### PR TITLE
add mark directory

### PR DESCRIPTION
**Pull request by Odrunia-Mark-Ryan**
pushing my exercise 14 files

### CODE REVIEW

File: Odrunia_Mark_Ryan/Countries/script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      displayRegionCountries(regionCountriesData);
+    } 
+    
+    catch (error) {
+      console.error('Error:', error);
+    }
+}
+  
+function displayCountryInfo(country) {
+  const countryInfoDiv = document.querySelector('#countryInfo');
```
Comment (JheanKhendrick) [inline Odrunia_Mark_Ryan/Countries/script.js:24]:
```suggestion
  const countryInfoDiv = document.querySelector('#country_info');
```

File: Odrunia_Mark_Ryan/Countries/script.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <p>Capital: ${country.capital}</p>
+    <p>Population: ${country.population}</p>
+    <p>Area: ${country.area} sq. km</p>
+    <p>Region: ${country.region}</p>
+    <!-- Add more details as needed -->
+  `;
+}
+
+function displayRegionCountries(countries) {
+  const regionCountriesDiv = document.querySelector('#regionCountries');
```
Comment (JheanKhendrick) [inline Odrunia_Mark_Ryan/Countries/script.js:39]:
```suggestion
  const regionCountriesDiv = document.querySelector('#region_countries');
```

File: Odrunia_Mark_Ryan/Countries/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        </div>
+        <div class="country-details">
+          <h2>Country Details</h2>
+          <div id="countryInfo"></div>
+        </div>
+      </div>  
+      <div class="box">
+        <div class="other-countries">
+          <h2>Countries in the Same Region: </h2>
+          <div id="regionCountries"></div>
```
Comment (JheanKhendrick) [inline Odrunia_Mark_Ryan/Countries/index.html:25]:
```suggestion
          <div id="region_countries"></div>
```

**Comment by Odrunia-Mark-Ryan** [comment]
issues fixed, also fixed my css id name for "searchInput to search_input", thank you for the review

**Review by JheanKhendrick — APPROVED** [review]

**Pull request merged**